microsoft-edge/devtools-guide-chromium/evaluate-performance/share-traces.md

```diff
@@ -0,0 +1,140 @@
+
+1. Click **Take Snapshot**.
+
+1. When the snapshot is recorded, click the **Save** button in the **Memory** tool sidebar.
+
+1. Choose a location to save the trace file on your disk.
+
+    ![Microsoft Edge with DevTools open, the Save button was clicked in the Memory tool, and an Explorer window is opened as a result, to save the exported file](./share-traces-images/exporting-memory-trace.png)
+
+Traces are exported as normal or enhanced traces, depending on the **Export Enhanced Performance and Memory Traces** option. To change the default trace type, see [Set the default trace type](#set-the-default-trace-type). You can also choose the type of trace you want when exporting.
+
+To choose a different trace type when exporting:
+
+1. Right-click (or hold the `Ctrl` key and click) the **Save profile** (![Save profile button](../media/export-icon.png)) in the **Performance** tool or the **Save** button in the **Memory** tool.
+
+1. Click either **.devtools (enhanced format for Microsoft Edge)** if you want to export an enhanced trace, or click **.json (works with Microsoft Edge and Chromium browsers)** if you want a normal trace.
+
+    ![The Performance tool, showing the contextual menu on the Save profile button, which allows to choose the type of trace](./share-traces-images/choose-trace-type-on-export.png)
+
+
+<!-- ====================================================================== -->
+## Import a trace
+
+To import a trace in the **Performance** tool:
+
+1. Open the **Performance** tool.
+
+1. Click **Load profile** (![Load profile button](../media/import-icon.png)).
+
+1. Locate the trace file on your disk. It's either a `.devtools` file (for enhanced traces) or a `.json` file (for normal traces).
+
+1. Open the file.
+
+    ![Microsoft Edge with DevTools open, the Load profile button was clicked in the Performance tool, and an Explorer window is opened as a result, to select a file to be imported](./share-traces-images/importing-perf-trace.png)
+
+    If the file is an enhanced trace, a new DevTools window appears with the performance profile and the extra runtime information that was recorded displayed in the **Sources**, **Console**, and **Elements** tools.
+
+    If the file is a normal trace, the performance profile appears in the **Performance** tool, and the rest of the DevTools tabs continue to show information related to the current web page.
+
+To import a trace in the **Memory** tool:
+
+1. Open the **Memory** tool.
+
+1. Click **Load**.
+
+1. Locate the trace file on your disk. It's either a `.devtools` file (for enhanced traces) or a `.heapsnapshot`, `.heaptimeline`, or `.heapprofile` file (for normal traces).
+
+1. Open the file.
+
+    ![Microsoft Edge with DevTools open, the Load button was clicked in the Memory tool, and an Explorer window is opened as a result, to select a file to be imported](./share-traces-images/importing-memory-trace.png)
```

Comment:
Global comment, esp applies to this png:

I wouldn't use two windows side-by-side, I'd design for narrow page instead; series of two windows, one below the other.  Probably as 2 pngs but could be 1 png containing two windows, one above the other, not side-by-side.  

GitHub rendered article w/ smaller headings than LMC:
https://github.com/MicrosoftDocs/edge-developer/blob/user/pabrosse/enhanced-traces/microsoft-edge/devtools-guide-chromium/evaluate-performance/share-traces.md


Right-click the bottom png, **Open in new tab**:
https://raw.githubusercontent.com/MicrosoftDocs/edge-developer/user/pabrosse/enhanced-traces/microsoft-edge/devtools-guide-chromium/evaluate-performance/share-traces-images/importing-memory-trace.png